test/Makefile.am

```diff
@@ -2,9 +2,10 @@
 # Makefile.am for storage/test
 #
 
-TESTS =									\
-	storage_snapper_configure_snapper_test.rb			\
-	storage_get_disk_partition.rb
+TESTS = \
+  storage_snapper_configure_snapper_test.rb \
+  storage_get_disk_partition.rb \
+  storage/snapshots_finish_test.rb
```

Comment:
Yes, it works. Just like this file: https://github.com/yast/yast-storage/blob/master/src/Makefile.am